build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+# Example: ./build.sh terra-jupyter-base
+
+IMAGE_DIR=$1
+
+TAG_NAME=$(git log --pretty=format:'%h' -n 1)
+docker image build ./$IMAGE_DIR --tag us.gcr.io/broad-dsp-gcr-public/$IMAGE_DIR:$TAG_NAME
+docker push us.gcr.io/broad-dsp-gcr-public/$IMAGE_DIR:$TAG_NAME
```

Comment:
This script must assume you're already authenticated with GCR?

Reply:
yea...the script is mainly used in jenkins. So auth part is done there. I'll add a comment in the script for people who run it locally